getting_started/first_3d_game/04.mob_scene.rst

```diff
@@ -172,24 +173,21 @@ between ``-PI / 4`` radians and ``PI / 4`` radians.
         RotateY((float)GD.RandRange(-Mathf.Pi / 4.0, Mathf.Pi / 4.0));
     }
 
-We then calculate a random speed using ``rand_range()`` once again and we use it
-to calculate the velocity.
-
-We start by creating a 3D vector pointing forward, multiply it by our
-``random_speed``, and finally rotate it using the ``Vector3`` class's
-``rotated()`` method.
+We got a random position, now we need a ``random_speed``. ``randi_range()`` will be useful as it gives random int values, and we will use ``min_speed`` and ``max_speed``.
+``random_speed`` is just an integer, and we just use it to multiply our ``CharacterBody3D.velocity``. After ``random_speed`` is applied, we rotate ``CharacterBody3D.velocity`` Vector3 towards the player.
 
 .. tabs::
  .. code-tab:: gdscript GDScript
 
    func initialize(start_position, player_position):
        # ...
 
-       # We calculate a random speed.
-       var random_speed = rand_range(min_speed, max_speed)
+       # We calculate a random speed (integer)
+       var random_speed = randi_range(min_speed, max_speed)
```

Comment:
This is wrong, this should be a float.

1. This value is always used in a float context.
2. It's perfectly sensible for a mob to have a speed of `10.5` meters per second for example.
3. This was float before, `rand_range` in Godot 3.x uses floats.

Reply:
I cannot re-open this now that it's merged, I suggest opening a new issue or doing a pull request, its pretty small anyway